src/vos/ilog.h

```diff
@@ -166,7 +166,7 @@ struct ilog_entry {
 	int32_t		ie_idx;
 };
 
-#define ILOG_PRIV_SIZE 408
+#define ILOG_PRIV_SIZE 600
```

Comment:
I plan to look into separating the slab array from the individual umm since it's the same for every pool.  This is remove the need to add 32 bytes for every slab entry here.  But that is for a different patch.